### ISSUE TITLE
[QUESTION]: can not do search when using a new database

### ISSUE THREAD

**Issue by MachineWei** [milvus-io/pymilvus#1715]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What is your question?

milvus-standalone : v2.2.13
pymilvus : 2.2.15

here is my code:

```python
from pymilvus import connections, db, utility
from pymilvus import Collection


connections.connect(host="localhost",
                    port="19530",
                    db_name="formal"
                    )

# db.using_database("formal")



# create collection
def create_collection():
    from pymilvus import FieldSchema, CollectionSchema, DataType
    id_field = FieldSchema(name="id", dtype=DataType.VARCHAR, is_primary=True, max_length=64,
                            description="primary id")
    text_field = FieldSchema(name="text", dtype=DataType.VARCHAR, max_length=65535, description="文本内容")
    embedding_field = FieldSchema(name="vector", dtype=DataType.FLOAT_VECTOR, dim=1024, description="vector")
    
    schema = CollectionSchema(fields=[id_field, text_field, embedding_field],
                              auto_id=False,
                              description='')
    
    collection = Collection(
        name='qa_1024',
        schema=schema,
        consistency_level="Strong",
        shards_num=2
    )
    # 建索引
    index_params = {
        "index_type": "HNSW",
        "metric_type": "L2",
        "params": {
            "M": 8,
            "efConstruction": 64
        }
    }
    collection.create_index(
        field_name="vector",
        index_params=index_params,
        index_name=""
    )
    # load进内存
    collection.load()

create_collection()

# search
search_params = {"metric_type": "L2", "params": {"ef": 10}}
col = Collection("qa_1024")
data = [[0.1]*1024]
result = col.search(
    data=data, 
    anns_field="vector", 
    param=search_params, 
    limit=10, 
    output_fields=['id', 'text', 'vector'], 
    consistency_level="Strong"
)
``` 
when i run it , i got error like this:
`
MilvusException: <MilvusException: (code=1, message=code: UnexpectedError, error: requery failed, err=code: UnexpectedError, reason: code: CollectionNotExists, reason: can't find collection: qa_1024)>
`

 

### Anything else?

when i release the collection,  i got :
`
MilvusException: <MilvusException: (code=1, message=checkIfLoaded failed when search, collection:qa_1024, partitions:[], err = GetCollectionInfo failed, collection = qa_1024, err = collection 444406101399309854 has not been loaded to memory or load failed)>
`

so , i confused

and i also try
```python
from pymilvus import MilvusClient

uri = "http://localhost:19530"
client = MilvusClient(uri, db_name='formal')

result = client.search(
    collection_name='qa_1024',
    data = data,
    limit=5,
    output_fields=['id', 'text', 'vector']
)

```
i got the same error

**Comment by XuanYang-cn** [comment]
Seems this bug has been fixed in commit 5870e5ef548dc462d626ca15a0a4aaae3a3b6ca7 
@MachineWei Please try pymilvus==2.2.16
/assign @MachineWei

**Comment by MachineWei** [comment]
Yes, thank you. it worked.
please update the tabel of "Milvus versions and recommended PyMilvus" in README.
